### DISCUSSION TITLE
Error message for each field not persist when using Livewire Forms

### DISCUSSION THREAD

**Discussion by nezaboravi** [livewire/livewire#7380] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Scenario is:

Contact form contains a few fields.

I want it to validate each field on blur.

If i enter wrong data in name field and error is displayed. This is expected.

I don't fix error, instead i tab to next field.

I enter wrong data in email field. I still see Name field error message. 

I tab away from Email field, and i see Email error, BUT, Name field error is suddenly gone.

I am expecting all the fields with error to persist, until i correct the entry.

Above validation will work when validation is done in LiveWire Component. 

When is extracted to Livewire Form, it does validate, but on every next error, it cleans up all other existing errors.

`php artisan livewire:form`

```
<?php

namespace App\Livewire\Forms;

use Livewire\Attributes\Validate;
use Livewire\Form;

class ContactForm extends Form
{
    #[Validate('required|min:5')]
    public string $name;
    #[Validate('required|email')]
    public string $email;
   
}
```

`php artisan make:livewire Contact`

In contact.blade.php

```
<section class="bg-white py-20 px-2" id="section-contact">
    <div class="container mx-auto px-4 lg:px-0 max-w-7xl"><h3 class="mb-10 text-center text-4xl font-semibold">Get in
            touch!</h3>
        <form class="mx-auto flex max-w-4xl flex-col gap-2" wire:submit="store">
            <div class="flex flex-col gap-2 py-2">
                <label for="contact_name">What's your name?</label>
                <input
                    wire:model.blur="form.name"
                    id="contact_name"
                    name="name"
                    type="text"
                    class="w-full rounded-lg border-none bg-blue-100">
                @error('form.name')
                {{ $message }}
                @enderror
            </div>
            <div class="flex flex-col gap-2 py-2">
                <label for="contact_email">Share your e-mail with us!</label>
                <input
                    wire:model.blur="form.email"
                    id="contact_email"
                    name="email"
                    type="email"
                    class="w-full rounded-lg border-none bg-blue-100">
                @error('form.email')
                {{ $message }}
                @enderror
            </div>          
            <div class="flex justify-center pt-10">
                <button type="submit" class="rounded-lg bg-blue-950 px-10 py-3 text-white">Send message</button>
            </div>
        </form>
    </div>
</section>
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by timmydhooghe** [comment]
Draft PR: #7644 
Temporarily downgrading to 3.1 should fix the issue, but I haven't verified that.

**Comment by luanfreitasdev** [comment]
I think they are different problems, but in this case, to avoid this behavior, you must set onUpdate: false

```php
class ContactForm extends Form
{
     #[Validate(rule: 'required|min:5', onUpdate: false)]
     public string $name;
     #[Validate(rule: 'required|email', onUpdate: false)]
     public string $email;
}
```
